src/openapi/peridio-admin-openapi.yaml

```diff
@@ -2102,6 +2103,72 @@ paths:
                 properties:
                   data:
                     $ref: "#/components/schemas/deployment"
+  /devices/{device_prn}/update:
+    get:
+      security:
+        - api_key_v2: []
+      operationId: devices-get-update
+      summary: devices get update
```

Comment:
This text looks inconsistent with the other operation summaries
<img width="303" alt="Screenshot 2025-01-30 at 12 29 13 PM" src="https://github.com/user-attachments/assets/df3a8972-3eb5-4f94-b3bf-d0a8f2e1eaf0" />

Should maybe be like `Get device update`